### PR TITLE
2015 & 2016 videos

### PR DESCRIPTION
**Pull request by MarijeP**
Still waiting on edited videos for Michael Gorven and Len Wencier. Marnus says he can do them next week.

@vhata can you make the video’s public to the whole world? Then this guy can be merged and we can tweet.

### CODE REVIEW

**Comment by adrianmoisey** [comment]
The videos can be viewed, as long as you have the URL for them

File: _data/speakers.yml [hunk 1]
```diff
@@ -112,6 +114,7 @@ bios:
           channels and will consider the IOT theme with a small example of a
           Lambda integration in to an Amazon Alexa for a simple beer
           ordering service :)
+        video: Ha2s_K9_91k
```
Comment (adrianmoisey) [inline _data/speakers.yml]: This is Michael Gorven, not Michael Needham

File: _data/speakers.yml [hunk 2]
```diff
@@ -326,6 +332,7 @@ bios:
           </ol>
           This talk will propose a model for scaling as well as discussing
           possible ways Etsy will continue to scale as it grows.
+        video: LfqyhM1LeIU
```
Comment (adrianmoisey) [inline _data/speakers.yml]: This is a link to `Bethany Macri - Retrospective: Scaling Infrastructure at Etsy`, not Michael's talk

File: _data/speakers.yml (hunk 1 above)
Comment (MarijeP) [inline _data/speakers.yml]: No repeating names, ever again. It clearly confuses Marije.

Comment (adrianmoisey) [inline _data/speakers.yml]: Definitely!

**Pull request merged**